controllers/cinderapi_controller.go

```diff
@@ -779,25 +777,48 @@ func (r *CinderAPIReconciler) generateServiceConfigs(
 	}
 	customData[cinder.CustomServiceConfigSecretsFileName] = customSecrets
 
-	// Inject Default logging: /dev/stdout doesn't work for cinder-api and a
-	// sidecar container is used to stream the logs to the defined LogPath.
-	// The following line makes the cinder-operator responsible to append to the
-	// ServiceConfig the logging line by default.
-	customData[cinder.CustomServiceConfigFileName] = fmt.Sprintf("%s\n%s%s\n",
```

Comment:
@Akrog instead of injecting an arbitrary snippet for Logging in `CustomServiceConfigFileName`, we're making the service able to use `templateParameters` and customize `00-config.conf` independently instead of inheriting it like the other services.
This is the basically the price to pay to make the service able to "diverge" a bit in its config according to the needs.
However, almost everything is still common among the services (which is the reason the `EnsureSecret` function has been called from `common.go`), but here we have the ability to specify a different `LogPath` if required without "hacking" any existing config file.